docs/how/restore-indices.md

```diff
@@ -0,0 +1,39 @@
+# Restoring Search and Graph Indices from Local Database
+
+If search or graph services go down or you have made changes to them that require reindexing, you can restore them from
+the aspects stored in the local database.
+
+When a new version of the aspect gets ingested, GMS initiates an MAE event for the aspect which is consumed to update
+the search and graph indices. As such, we can fetch the latest version of each aspect in the local database and produce
+MAE events corresponding to the aspects to restore the search and graph indices.
+
+## Docker-compose
+
+Run the following command from root to send MAE for each aspect in the Local DB.
+
+```
+./docker/datahub-upgrade/datahub-upgrade.sh -u RestoreIndices
+```
+
+If you need to clear the search and graph indices before restoring, add `-a clean` to the end of the command.
```

Comment:
Consider adding a note about the default env variables used and how to change them to suit your deployment

Reply:
Added a link to the datahub-upgrade readme section on env variables. Also updated that doc